DSharpPlus/Enums/Message/MessageType.cs

```diff
@@ -99,7 +99,7 @@ public enum MessageType : int
         GuildDiscoveryDisqualified = 14,
 
         /// <summary>
-        /// Message indicating a guild was readded to guild discovery.
+        /// Message indicating a guild was readed to guild discovery.
```

Comment:
This was correct before. Maybe change it to `re-added` for clarity.

Reply:
sure!